### DISCUSSION TITLE
broken response body string in log

### DISCUSSION THREAD

**Discussion by daewooLeeNy** [mockoon/mockoon#472] [Help] [answered]
I am using proxy mode in my local.

But it looks like broken string in response body viewer.
see the below image:

<img width="785" alt="스크린샷 2021-05-07 오후 1 08 37" src="https://user-images.githubusercontent.com/12162890/117397277-b5e96a00-af36-11eb-8684-4f0226b8be6a.png">

How do i show normally response string?

Thank you for your work

**Comment by 255kb** [comment]
Hi, you should be able to view string bodies without problem. You usually see this for binary bodies (png, etc.) as this view only show a stringified version of the raw body. 
For example, I just tested to proxy to `https://petstore.swagger.io/` and I could see the JSON returned from Swagger:
![image](https://user-images.githubusercontent.com/7489814/117414479-74959200-af17-11eb-82ba-5bf99865bcf4.png)

To check if there is a bug, could you please provide enough info so I can try to reproduce?

**Comment by barefootdeveloper** [comment, marked as answer]
@daewooLeeNy Your response looks weird here because the server that is fulfilling your proxied request has gzip compression turned on. You can see it in the response headers content-encoding: gzip.  

This relates to the discussion here: https://github.com/mockoon/mockoon/discussions/453 and would make for a very useful featured to add to mockoon.

**Reply by 255kb** [reply]
Indeed, it must be linked to the gzip compression. I guess the problem occurs only when receiving a gzipped response from a proxied API. As we cannot interact in any way with such response in Mockoon, the only place where unzipping would have an effect would be in the logs body preview.
Could you please open an issue to track this @daewooLeeNy ? 

So I guess this discussion is now solved.

**Reply by 255kb** [reply]
An issue has been opened #479
